src/hardware/voodoo.cpp

```diff
@@ -7805,8 +7913,11 @@ void VOODOO_Init(Section* sec)
 	PCI_AddDevice(new PCI_SSTDevice());
 
 	// Log the startup
-	LOG_MSG("VOODOO: Initialized with %s MB of RAM, %smultithreading, and %sbilinear filtering",
+	const auto num_threads = get_num_threads();
+
+	LOG_MSG("VOODOO: Initialized with %s MB of RAM, %d %s, and %sbilinear filtering",
 	        memsize_pref.c_str(),
-	        (voodoo_multithreading ? "" : "no "),
+	        num_threads,
+	        num_threads == 1 ? "thread" : "threads",
```

Comment:
Nice touch 😎

Reply:
thanks :-)
OK.. sending the PR. Thanks for the wrap-up comments.